v_3_0_0/master_template.go

```diff
@@ -452,9 +452,21 @@ write_files:
               effect: NoSchedule
             - key: "CriticalAddonsOnly"
               operator: "Exists"
+          affinity:
+            podAntiAffinity:
+              preferredDuringSchedulingIgnoredDuringExecution:
```

Comment:
Why not required? Does it make sense to run 2 coredns on the same machine?

Reply:
Can’t say for now. I tested antiaffinity with required and it fails to run even one pod. 
Soft requirement used also in nginx ic

Reply:
Ack.

Reply:
It does make sense even on the same machine as they cover load no matter on what machine. It would be best to have them on more machines for failover reasons.